go/vumitools/tests/test_billing_worker.py

```diff
@@ -45,7 +46,8 @@ def create_transaction(self, account_number, message_id, tag_pool_name,
             "credit_factor": decimal.Decimal('0.4'),
             "created": "2013-10-30T10:42:51.144745+02:00",
             "last_modified": "2013-10-30T10:42:51.144745+02:00",
-            "status": "Completed"
+            "status": "Completed",
+            "transaction_type": BillingDispatcher.TRANSACTION_TYPE_MESSAGE
```

Comment:
Seems weird using `BillingDispatcher` from the dummy billing api. Makes me think `TRANSACTION_TYPE_MESSAGE` should be at the module level, but that would be inconsistent, at the moment it is on `BillingDispatcher` since `MESSAGE_DIRECTION_INBOUND` and `MESSAGE_DIRECTION_OUTBOUND` are.